gymnasium/envs/classic_control/mountain_car.py

```diff
@@ -39,20 +39,18 @@ class MountainCarEnv(gym.Env):
 
     The observation is a `ndarray` with shape `(2,)` where the elements correspond to the following:
 
-    | Num | Observation                          | Min  | Max | Unit         |
-    |-----|--------------------------------------|------|-----|--------------|
-    | 0   | position of the car along the x-axis | -Inf | Inf | position (m) |
-    | 1   | velocity of the car                  | -Inf | Inf | position (m) |
+    | Num | Observation                          | Min   | Max  | Unit         |
+    |-----|--------------------------------------|-------|------|--------------|
+    | 0   | position of the car along the x-axis | -1.2  | 0.6  | position (m) |
+    | 1   | velocity of the car                  | -0.07 | 0.07 | velocity (v) |
 
     ## Action Space
 
     There are 3 discrete deterministic actions:
 
-    | Num | Observation             | Value | Unit         |
-    |-----|-------------------------|-------|--------------|
-    | 0   | Accelerate to the left  | Inf   | position (m) |
-    | 1   | Don't accelerate        | Inf   | position (m) |
-    | 2   | Accelerate to the right | Inf   | position (m) |
+    - 0: Accelerate to the left
```

Comment:
Why have you changed the table to the list

Reply:
As the action is discrete I founded more understandable.

The action itself doesn't have any `Value` of `Unit`, thus I followed the same format as for Frozen Lake.

Reply:
That is a good point, could you similarly update cartpole, car racing, lunar lander, blackjack

Reply:
sure 👍

Reply:
done

Reply:
Thanks